src/dune_rules/lib_id.mli

```diff
@@ -1,13 +1,26 @@
 open Import
 
+type visibility =
+  | Public of Public_lib.t
+  | Private of Package.t option
+
 module Local : sig
   type t
 
   module Map : Map.S with type key = t
   module Set : Set.S with type elt = t
 
   val equal : t -> t -> bool
-  val make : loc:Loc.t -> src_dir:Path.Source.t -> enabled_if:Blang.t -> Lib_name.t -> t
+
+  val make
+    :  loc:Loc.t
+    -> src_dir:Path.Source.t
+    -> enabled_if:Blang.t
+    -> visibility:visibility
+    -> Lib_name.t
+    -> t
+
+  val best_name : t -> Lib_name.t
```

Comment:
where is this function used?

Reply:
not needed. artifact of a previous version